cetmix_tower_yaml/tests/test_server_template.py

```diff
@@ -0,0 +1,16 @@
+from odoo.tests import TransactionCase
+
+
+class TestTowerServerTemplate(TransactionCase):
+    def setUp(self, *args, **kwargs):
+        super().setUp(*args, **kwargs)
+
+        self.ServerTemplate = self.env["cx.tower.server.template"]
+
+    def test_server_template_ssh_password_mask(self):
+        """Test that SSH password is masked in YAML"""
+        server_template = self.ServerTemplate.create(
```

Comment:
It is recommended to add exception handling when creating the `ServerTemplate`. This will make the test more robust and help diagnose issues more easily.

Reply:
Good notice, thank you! However there is already  an exception handler [implemented](https://github.com/cetmix/cetmix-tower/blob/6c43da83327f3d6c917419759c445bbba888e2d5/cetmix_tower_yaml/models/cx_tower_yaml_mixin.py#L84) at the mixin level.
Which is being tested [here](https://github.com/cetmix/cetmix-tower/blob/6c43da83327f3d6c917419759c445bbba888e2d5/cetmix_tower_yaml/tests/test_tower_yaml_mixin.py#L411)